src/leaflet.plotter.js

```diff
@@ -20,7 +23,7 @@ L.Polyline.plotter = L.Polyline.extend({
             this._bindMapClick();
         }
     },
-    onRemove: function(){
+    onRemove: function(map){
```

Comment:
No, just look at line 32 in your sources or 35th in my fork.
You use `map` variable but it is not defined in function scope
```js
     L.Polyline.prototype.onRemove.call(this, map);
```